kafka/src/main/java/org/apache/apex/malhar/kafka/AbstractKafkaInputOperator.java

```diff
@@ -255,6 +255,10 @@ public void emitTuples()
       }
     }
     emitCount += count;
+
+    if (!consumerWrapper.doesAnyKafkaReaderThreadDied()) {
```

Comment:
If any of these thread dies because of the exception, how to handle that cleanly?